rpm/3.4/arangodb3e.spec.in

```diff
@@ -232,13 +235,13 @@ exit 0
 %pre
 # create arangodb group only if it doesn't already exist
 if ! getent group arangodb >/dev/null 2>&1; then
-        /usr/sbin/groupadd -r arangodb
+  /usr/sbin/groupadd -r arangodb
 fi
 
 # create arangodb user only if it doesn't already exist
 if ! getent passwd arangodb >/dev/null 2>&1; then
-        /usr/sbin/useradd -r -g arangodb --home %{_localstatedir}/arangodb arangodb
-        /usr/sbin/usermod -c "Arango Server" arangodb
+  /usr/sbin/useradd -r -g arangodb --home %{_localstatedir}/arangodb arangodb
```

Comment:
are the RPM distributions also moving into the direction of putting everything to one directory, as we saw with the debian docker containers? Or should we first have a `test -f` and else to `/sbin/chmod ` etc?